packages/rs-dpp/src/data_trigger/feature_flags_data_triggers/mod.rs

```diff
@@ -41,15 +43,15 @@ where
         )
     })?;
 
-    let latest_block_header: JsonValue = context
+    let core_chain_locked_height = context
         .state_repository
-        .fetch_latest_platform_block_header()
-        .await?;
+        .fetch_latest_platform_core_chain_locked_height()
```

Comment:
@antouhou please check whether this substitution is acceptable

Reply:
Yep, here too